test/JuMP_sparse_linear_mpc.jl

```diff
@@ -0,0 +1,231 @@
+using Ipopt, JuMP, Random, LinearAlgebra
+
+
+
+"""
```

Comment:
it's good to have the docstring for this, but not particularly necessary since we are using this function just for testing. Next time don't spend too much time on docstrings for testing

Reply:
Does this apply to internal functions that are not being exported as well, such as `_build_J` or `_build_H`?